TreeMaker/python/maker.py

```diff
@@ -138,9 +149,11 @@ def printSetup(self):
         print " Instance name of tag information: "+self.tagname
         print " Instance name of HLT tag information: "+self.hlttagname
         print " Including gen-level information: "+str(self.geninfo)
-        print " Including pMSSM-related information: "+str(self.pmssm)
         print " Using fastsim settings: "+str(self.fastsim)
         print " Running signal uncertainties: "+str(self.signal)
+        if len(self.includeBranches)>0: print " Including branches: "+','.join(self.includeBranches)
+        if len(self.excludeBranches)>0: print " Excluding branches: "+','.join(self.excludeBranches)
+        if self.exactBranches: print "  Using exact branches (no regex)"

```

Comment:
I feel like there should be an easier way to automatically detect if something is a regex, but I can't think of it right now. In general, it would be nice to not have a manual switch and to be able to mix exact patterns with regex.

Reply:
Again, not sure I see the motivation for such a complicated use case at this time.